templates/certification/index.html

```diff
@@ -3,40 +3,237 @@
 {% block title %}Certified hardware | Ubuntu{% endblock %}
 
 {% block meta_description %}Hardware that have been certified for use with Ubuntu.{% endblock meta_description %}
-{% block meta_copydoc %}https://docs.google.com/document/d/1QuhO-9FEOGLrYp8bErS_9snqdljl7d6tFAUoNQxoVDQ/edit{% endblock meta_copydoc %}
+{% block meta_copydoc %}{% endblock meta_copydoc %}
```

Comment:
I am not 100% but does [mpt's doc](https://docs.google.com/document/d/1vIf1vC1mg72JbzmahrcKCaRZN-qMLCgin_i9iqjUxpM/) count as a copy doc? If not I think we can remove the entire block. @anthonydillon what do you think?

Reply:
I think it needs to be a copy doc in the correct format which I think will be done by maybe @anasereijo or @SirSamTumless who did we say should do it? I left the block there to remind me to chase this

Reply:
It's not perfect but something is better than nothing. I would say keep what we got and ask whoever updates the doc to update the existing one.